metadata/base/metadata-envoy-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: metadata-envoy
+  labels:
+    component: metadata-envoy
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: metadata-envoy
+  template:
+    metadata:
+      labels:
+        component: metadata-envoy
+    spec:
+      containers:
+      - name: container
```

Comment:
Kubeflow deploys istio envoy. Do we need a dedicated envoy for metadata?

Reply:
@zhenghuiwang good point. However, the UI will need to know how to proxy requests to the MLMD grpc server, and that config will need to be in sync between KFP and KF. My suggestion is as follows: we use this for now, but get rid of this completely in Q4 in favour of a HTTP API on top of the MLMD server. What do you think? We can also wait to complete that before updating it this way, but we need this to work now so UI folks can build the lineage parts.

Reply:
I'm okay with having a dedicated metadata envoy for now and getting rid of it later.